packages/template-retail-react-app/app/components/item-variant/item-price.jsx

```diff
@@ -32,8 +26,7 @@ const PricePerItem = ({currency, basket, basePrice}) => {
 }
 
 PricePerItem.propTypes = {
-    currency: PropTypes.string,
-    basket: PropTypes.object,
+    currency: PropTypes.string.isRequired,
```

Comment:
What is this component? Its called "PricePerItem" but I find it hard to think of what it's used for based on that name? Is it even big enough to be its own component? 

Sorry about commenting on code that isn't really directly related to your change, but since we are making a breaking change...

Reply:
It is the component to show price of a single item when the line item has item quantity > 1. I think it is fine to be its own component

Reply:
There is a future initiative to make the template easier to override by not having multiple components per file. You don't have to do it in this pr.. but know know in the future that we'll probably be moving that component.